### ISSUE TITLE
Undeclared dependency Carp::Always

### ISSUE THREAD

**Issue by eserte** [hoelzro/devel-trace-syscall#18]
t/run_tests.t fails if Carp::Always is not installed:
```
Can't locate Carp/Always.pm in @INC (you may need to install the Carp::Always module) (@INC contains: /tmpfs/.cpan-build-cpansand/2017052903/Devel-Trace-Syscall-0.03-GGsLim/blib/lib /tmpfs/.cpan-build-cpansand/2017052903/Devel-Trace-Syscall-0.03-GGsLim/blib/arch /home/cpansand/.cpan/build/2017052903/Test-Differences-0.64-OOLbgq/blib/arch /home/cpansand/.cpan/build/2017052903/Test-Differences-0.64-OOLbgq/blib/lib /home/cpansand/.cpan/build/2017052903/Test-Differences-0.64-OOLbgq/blib/arch /home/cpansand/.cpan/build/2017052903/Test-Differences-0.64-OOLbgq/blib/lib /opt/perl-5.20.3D/lib/site_perl/5.20.3/x86_64-linux-ld /opt/perl-5.20.3D/lib/site_perl/5.20.3 /opt/perl-5.20.3D/lib/5.20.3/x86_64-linux-ld /opt/perl-5.20.3D/lib/5.20.3 .) at t/run_tests.t line 8.
BEGIN failed--compilation aborted at t/run_tests.t line 8.
t/run_tests.t ... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by hoelzro** [comment]
D'oh!  Fixing this up now

**Comment by hoelzro** [comment]
This is fixed in 0.04
